new/LIT6193HomiliaryPassions.xml

```diff
@@ -0,0 +1,155 @@
+               ርቱዓ፡ ዘአማዘነ፡ መንበሮ፡
+             </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyMonday">
+             <label>Homily for Holy Monday</label>
+             <ab>
+               <pb n="7r"></pb>
+               <cb n="a"></cb>ናንቅሕ፡ ርእሰ፡ ሕሊናነ፡ ወናስተሐውስ፡ አስራወ፡ ልሳንነ፡ ንክሥት፡ አፉነ፡ ወናስተናብብ፡ ከናፍሪነ፡
+             </ab>
+             </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyTuesday">
+          <label>Homily for Holy Tuesday</label>
+          <ab>
+            <pb n="14v"></pb>
+            <cb n="b"></cb><hi rend="rubric">ንግባእኬ፡ ከመ፡ ንፈጽም፡ ዜና፡ ሕማሙ፡</hi>
+             ወአንግሁ፡ ይእተ፡ ኵላ፡ ሊሊት፡ እንዘ፡ ይዘብጥዎ፡ ወይጸፍዕዎ
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyWednesday">
+          <label>Homily for Holy Wednesday</label>
+          <ab>
+            <pb n="19v"></pb>
+            <cb n="a"></cb><hi rend="rubric">ንግባዕኬ፡ ከመ፡ ንፈጽም፡ ነገረ፡ ወበጊዜ፡</hi>
+            ፱ሰዓት፡ ተከሥተ፡ ፀሐይ፡ በዓውደ፡ ሰማይ፨
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyThursday">
+          <label>Homily for Maundy Thursday</label>
+          <ab>
+            <pb n="25v"></pb>
+            <cb n="b"></cb><hi rend="rubric">ንግባዕኬ፡ ከመ፡ ንፈጽም፡ ነገረ፡ ዘ፱ሰዓት፡</hi>
+            ወይእተ፡ ጊዜ፡ ሶበ፡ ርእዩ፡ ነቢያት፡ ቀደምት፡ ወአበው፡ ቀደምት፡
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyFriday">
+          <label>Homily for Good Friday</label>
+          <ab>
+            <pb n="29v"></pb>
+            <cb n="b"></cb><hi rend="rubric">ንግባዕኬ፡ ከመ፡ ንፈጽም፡ ነገረ፡ ዘምሴተ፡</hi>
+            ዓርበ፡ እስመ፡ ይቤ፡ ዘካርያስ፡ ይእቲ፡ ዕለት፡ እምብርት፡ በኀበ፡ እግዚአብሔር፡ ኢኮነት፡
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilySaturday">
+          <label>Homily for Holy Saturday</label>
+          <ab>
+            <pb n="33r"></pb>
+            <cb n="a"></cb><hi rend="rubric">ንግባዕኬ፡ ከመ፡ ንፈጽም፡ ነገረ፡ ዘምሴተ፡</hi>
+            ዓርበ፡ እስመ፡ ይቤ፡ ዘካርያስ፡ ይእቲ፡ ዕለት፡ እምብርት፡ በኀበ፡ እግዚአብሔር፡ ኢኮነት፡
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilySunday">
+          <label>Homily for Passion Sunday</label>
+          <ab>
+            <pb n="35r"></pb>
+            <cb n="a"></cb>ንግባዕኬ፡ ከመ፡ ንሰብሖ፡ ወንወድሶ፡ ለእግዚአብሔር፡
+            እንዘ፡ ንብል፡ ጥዑመ፡ ስም፡ መድኃኒነ፡ ለሊሁ፡ ንጉሥ፡ ወእሙ፡ መቅደስ፡ ወመስቀሉ፡ አትሮንስ፡
+          </ab>
+          </div>
+
+          <div type="textpart" subtype="chapter" xml:id="HomilyConclusion">
+          <label>Concluding supplication</label>
+          <ab>
+            <pb n="36r"></pb>
+            <cb n="a"></cb>ለዘጸሐፎ፡ ወለዘአጽሐፎ፡ ለዝንቱ፡ ድርሳን፡ እግዚአብሔር፡ ይጽሐፍ፡ ስሞ፡ በአምደ፡<cb n="a"></cb>
+             ብርሃን፡
+          </ab>
+          </div>
+          </div>
+
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6193HomiliaryPassions" passive="BNFabb29"/>
```

Comment:
this belongs to the listWit not relation

Reply:
Someone told me, in a very certain way, that listWit is used only for (digital) edition. I always included witness mss into this field, i think the others as well.

Reply:
listWit should only be used for manuscripts used for an edition published in the same record. As there were different guidelines at the beginning of BM, there is no need to update the old records, but this should not be replicated. However, I would also not use the hasAttestation relation here (though not wrong, no need to go back and change the record now) - as long as the work ID is referenced in the manuscript, it will be picked up by the app.